src/plugin/Manager.cc

```diff
@@ -160,6 +162,9 @@ void Manager::SearchDynamicPlugins(const std::string& dir)
 bool Manager::ActivateDynamicPluginInternal(const std::string& name, bool ok_if_not_found,
                                             std::vector<std::string>* errors)
 	{
+#if defined(_MSC_VER)
+	return false;
```

Comment:
Add a comment here about why this function doesn't work on Windows.

Reply:
done